stout_a06c.js

```diff
@@ -0,0 +1,42 @@
+/*Kyle Stout
+  stout_a06c
+  INFO 2124WW
+  Thoendel
+  1-26-2020
+*/
+
+
+//winning numbers: 22, 42, 43, 47, 61;
+
+let lottoPicks = [];
+let powerBallPick = getLottoNumber(); 
+let winnings = 0; 
+
+/* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+let count = 0;
+//initialize count for filling array
+while (count < 5){
+//start the while loop
+lottoPicks.push(getLottoNumber());
+//push random value of function getLottoNumber into array
+count++;
+//up the count so that I will end up with only 5 values
+}
+if lottoPicks[].includes(22) {
+    console.log("Matched 22")
+} else {
+    console.log("")
+}

```

Comment:
here I would expect a for loop to loop through lottoPicks[i] and evaluate its value with a switch statement. Let me know if you have further questions - we could meet during my assistance hours to go over it.
-10

Reply:
I actually tried to do that but I couldn't get a switch statement inside a for loop to validate my inner java wants to believe there is a scope issue but I honestly don't know how to fix that.

Reply:
If you want to give it a go - I'll take a look and let you know how to get it working. It wouldn't be worth any points - but it is a skill you will need going forward. Let me know how I can help.

Reply:
When and where are your office hours? I really want to learn javascript but I'm more confused then anything as we progress in this course.